django/db/models/functions/text.py

```diff
@@ -70,6 +70,17 @@ class Lower(Transform):
     lookup_name = 'lower'
 
 
+class Replace(Func):
+    function = 'REPLACE'
+    name = 'Replace'
+
+    def __init__(self, expression, text, replacement='', **extra):
+        if not hasattr(replacement, 'resolve_expression'):
+            replacement = Value(replacement)
```

Comment:
Not sure we should be special case `replacement` direct string passing. We usually consider strings as field references.

Reply:
Yes, that way it'll also be more in style with PR #8272. I'll update to have `V('')` or `Value('')` as the default.